assets/src/js/components/settings.js

```diff
@@ -42,10 +42,10 @@ function Component() {
                     <div className={"grid"} style={"margin-bottom: 24px;"}>
                         <div style={"grid-column: span 4;"}>
                             <h1>{i18n['Settings']}</h1>
-                            <form method={"POST"} onsubmit={onsubmit}>
+                            <form method={"POST"} onSubmit={onSubmit}>
```

Comment:
Looks like your editor is changing this automatically, but unfortunately `onSubmit` & `onChange` breaks the application as Mithril.js uses lowercased event attributes.